.github/workflows/validate.yml

```diff
@@ -0,0 +1,29 @@
+name: Validate
+on: push
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
@rosslo could you explain why need these 2 steps: `actions/checkout@v2` and `actions/cache@v1`

Reply:
actions/checkout@v2: to help us `checkout` the latest pushed commit, otherwise, we will need to do a few git commands to implement this. Can see more on the official document: https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#using-the-checkout-action

actions/cache@v1: to save the yarn install time, after experiments, I found it saves 2x seconds.

Reply:
thank u @rosslo got it!